src/transforms/globalCssToCssModule/postcss/__tests__/transformFileToCssModule.test.ts

```diff
@@ -5,6 +5,8 @@ const replaceWhitespace = (value: string) => value.replace(/\s+/g, ' ').trim()
 describe('transformFileToCssModule', () => {
     it('correctly transforms provided CSS to CSS module', async () => {
         const sourceCss = `
+            @import './RepositoriesPopover';
```

Comment:
@valerybugakov What is about some global styles like reach-ui components styles? Do they move to tsx files?

Reply:
Generally, [we should not have global styles](https://sourcegraph.slack.com/archives/C01C3NCGD40/p1629356141113900) in component related stylesheets. All reach-ui styles were extracted into the global styles entry-point. If you know of any other cases in our codebase, let me know.